src/components/_proxied-dot-io/packer/remote-plugin-docs/server.ts

```diff
@@ -146,9 +146,14 @@ async function generateStaticProps({
 	const content = await mdxContentHook(rawContent)
 
 	// Render MDX source, with options
+	const headings = [] // populated by anchorLinks plugin below
```

Comment:
The existing dot-io implementation relies on [@hashicorp/react-docs-page's client-side anchor links shim](https://github.com/hashicorp/react-components/blob/main/packages/docs-page/temporary_jump-to-section.js).

For Dev Dot, we want to use our `anchorLinks` plugin's `headings` option, to allow us to construct `headings` server-side. I've added this to the shared utility (rather than trying to shim it in _only_ in dev dot contexts) because there's no real harm for the dot-io implementation.